src/Engine/ProtoCore/FFI/CLRObjectMarshaler.cs

```diff
@@ -1381,7 +1379,7 @@ void core_Dispose(ProtoCore.RuntimeCore sender)
     /// </summary>
     public class ReferenceEqualityComparer: IEqualityComparer<object>
     {
-        public bool Equals(object x, object y)
```

Comment:
You cannot just remove the public access modifier. This would be a breaking change

Reply:
There are just two possibilities to fix this issue, to define a new member hiding the object.Equals implementation, or implement explicit the member Equals also defined in the interface IEqualityComparer<object>, which one do you think fits better in this context?
![image](https://github.com/DynamoDS/Dynamo/assets/132492346/f7f5e7ec-3f75-43dc-aaf7-0f63a91d3b31)